pkg/gameserverallocations/controller.go

```diff
@@ -398,26 +398,33 @@ func (c *Controller) allocateFromRemoteCluster(gsa v1alpha1.GameServerAllocation
 	}
 
 	// TODO: Retry on transient error --> response.StatusCode >= 500
-	response, err := client.Post(connectionInfo.AllocationEndpoint, "application/json", bytes.NewBuffer(body))
-	if err != nil {
-		return nil, err
-	}
-	defer response.Body.Close() // nolint: errcheck
+	for i, endpoint := range connectionInfo.AllocationEndpoints {
```

Comment:
Should we be randomising the endpoints? (or this going to be policy driven in the future?)

Reply:
This is requesting based on the order of endpoints set in the policy. I wouldn't randomize it to have a consistent behavior, but I also see the value of choosing the endpoint that works first. It is good for now. 

For future, we can update policies endpoints based on working endpoints, but it needs more thinking. WDYT?

Reply:
SGTM :+1: